test/contracts/apps/disputable/disputable_app.js

```diff
@@ -122,7 +126,11 @@ contract('DisputableApp', ([_, owner, agreement, anotherAgreement, someone]) =>
       const from = someone
 
       it('reverts', async () => {
-        await assertRevert(disputable.setAgreement(agreement, { from }), 'APP_AUTH_FAILED')
+        await assertRevert(dao.setAgreement(disputable.address, agreement, { from }), 'KERNEL_AUTH_FAILED')
+      })
+
+      it('reverts', async () => {
```

Comment:
We can probably come up with better test names, since these two are not the same test case :)